crates/wasm-smith/src/component.rs

```diff
@@ -628,9 +628,11 @@ impl ComponentBuilder {
             });
         }
 
-        if for_type_def || !for_import || self.total_values < self.config.max_values() {
-            choices.push(|me, u| Ok(ComponentTypeRef::Value(me.arbitrary_component_val_type(u)?)));
-        }
+        // TODO: wasm-smith needs to ensure that every arbitrary value gets used exactly once.
```

Comment:
@fitzgen here's the disabling of arbitrary value imports I pinged you about for your review.